### PR TITLE
Add Codecove to the transport-http repo

### PR DESCRIPTION
**Pull request by SmithAbey**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,3 +20,8 @@ cache:
 
 jdk:
   - openjdk8
+  
+script: "mvn cobertura:cobertura"
+
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```
Comment (ldclakmal) [inline .travis.yml:27]:
```suggestion
  - bash <(curl -s https://codecov.io/bash)
  
```

File: components/org.wso2.transport.http.netty/src/test/resources/testng.xml [hunk 2]
```diff
@@ -37,7 +37,7 @@
             <class name="org.wso2.transport.http.netty.contentaware.RequestResponseTransformStreamingTestCase" />
 
             <class name="org.wso2.transport.http.netty.proxyserver.HttpsProxyServerTestCase" />
-            <class name="org.wso2.transport.http.netty.proxyserver.HttpProxyServerTestCase" />
+            <class name="org.wso2.transport.http.netty.proxyserver.HttpProxyServerTestCase"/>
```
Comment (ldclakmal) [inline components/org.wso2.transport.http.netty/src/test/resources/testng.xml:40]:
```suggestion
            <class name="org.wso2.transport.http.netty.proxyserver.HttpProxyServerTestCase" />
```

File: pom.xml [hunk 3]
```diff
@@ -290,6 +290,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>cobertura-maven-plugin</artifactId>
+                <version>2.7</version>
```
Comment (ldclakmal) [inline pom.xml:296]: Shall we parameterize this version as we did in other plugins?
```suggestion
                <version>2.7</version>
```

**Review by Kishanthan — COMMENTED** [review]
I believe @SmithAbey has to sign CLA here

**Pull request closed without merging**